subsys/net/lib/aws_fota/src/aws_fota_json.c

```diff
@@ -36,19 +47,12 @@ struct execution_obj {
 };
 
 struct notify_next_obj {
+	const char *client_token;
```

Comment:
How could this work prior to this change?

Reply:
Because the execution object does not always return status details. It worked for parsing some objects.